config/settings.yml.example

```diff
@@ -58,6 +58,11 @@
 # default values for https_port is 8443
 #:https_port: 8443
 
+# http server type configuration
+# A string that indicates the server type (possible values: 'webrick', 'puma').
+# Default value is 'webrick'
+#:http_server_type: 'puma'
```

Comment:
Btw. while working on the cockpit integration and using proxy to tunnel the ssh connection, we've found Puma having much nicer support for that stuff.

Would we plan to still keep webrick in 1.22, or would the puma be already the new default? It influences us going forward with cockpit, as with webrick, we would still need to keep the workaround so that it works with it.

Reply:
Well don't ask me, you are the reviewer here :-)

Honestly I think that 1.22 would be opt-in only, but what stops us from saying "If you want cockpit integration you must switch to Puma"? We can probably expose the setting via features to check for that in the core app.

Reply:
I would like to avoid depending on non-default: we will see how hard it will be to get the webrick hacks working with cockpit

Reply:
I don't see a reason why not to try this and only if it does not work you can then investigate other paths. I am currently working on SD notify and this should be ready for final review.

Reply:
The reason is I don't want the cockpit users to be exploring types of deployment, that other uses would not be running because it's not default. The more options there are to setup, the worse. Unless it's not default, I don't want the cockpit integration to depend on it

Reply:
I am not against making it default, users can fallback. Let's do it.